src/components/tasks/card/ShowError.tsx

```diff
@@ -0,0 +1,7 @@
+import Image from 'next/image';
+import classNames from '@/components/tasks/card/card.module.scss';
+export const ShowError = () => (
+    <div className={classNames.selfAlignEnd} data-testid="checkmark">
+        <Image src="/error.png" alt="data saved" width={20} height={20} />
```

Comment:
the alt tag seems a little odd here. can be something else

Reply:
I have Updated it now.